WordPress/Classes/ViewRelated/NUX/SiteCreationNavigationController.swift

```diff
@@ -1,7 +1,30 @@
 import UIKit
 
+@objc protocol SiteCreationNavigationControllerDelegate {
```

Comment:
I'm reiterating my big comment above, but I think we should skip this delegate as well. Unless I'm mistaken and there's a technical reason?